src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/support/SupportController.java

```diff
@@ -156,4 +178,32 @@ private void resendStaffNotificationOnAgreementCountersigned(Claim claim) {
         agreementCountersignedStaffNotificationHandler.onAgreementCountersigned(event);
     }
 
+    private void resendClaimsToRPA(List<Claim> claims, String authorisation) {
+        if (StringUtils.isBlank(authorisation)) {
+            throw new BadRequestException("Authorisation is required");
+        }
+
+        for (Claim claim: claims) {
+            documentGenerator.generateForNonRepresentedRPA(
+                new SupportClaimIssuedEvent(claim, authorisation)
+            );
+        }
+    }
+
+    private List<Claim> checkClaimsExist(List<String> referenceNumbers) {
+        if (referenceNumbers.isEmpty())
+        {
+            throw new NullPointerException("Reference Numbers not supplied");
```

Comment:
More of illegal argument exception than null pointer if empty list is provided. In fact such checks to be done right up at the point of entry of endpoint.